source/Ship.cpp

```diff
@@ -807,7 +824,9 @@ string Ship::FlightCheck() const
 		if(fuelCapacity < JumpFuel())
 			return "no fuel?";
 	}
-	
```

Comment:
Looks like your editor ate indentation

Reply:
The energy check is a warning condition, so I put it in the same paragraph as the other ones.

Reply:
Yes, and the newline you added after it is missing indentation. Thus, GitHub shows you as also removing this indented line, instead of only adding the new lines. For a small pull it's easy to fix but as you edit more files you may want to find the IDE setting that governs this behavior and configure it for this repo.